tools/monitoring/mail/mail.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mail
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/smtp"
+	"strings"
+)
+
+const (
+	smtpHost = "smtp.gmail.com"
+	smtpPort = "587"
```

Comment:
should this be port 465?

Reply:
I was able to send an email with this port. 465 is the SSL port, 587 is the TLS/STARTTLS port. 

https://support.google.com/mail/answer/7126229?hl=en

Reply:
👍